gravitee-apim-console-webui/src/management/api/api-traffic-v4/analytics/api-analytics-widget.service.ts

```diff
@@ -114,13 +123,22 @@ export class ApiAnalyticsWidgetService {
   private handleStatsAnalytics$(
     widgetConfig: ApiAnalyticsDashboardWidgetConfig,
     timeRangeParams: TimeRangeParams,
+    httpStatuses?: string[],
+    plans?: string[],
+    hosts?: string[],
+    applications?: string[],
   ): Observable<ApiAnalyticsWidgetConfig> {
```

Comment:
```suggestion
  private handleStatsAnalytics$(
    widgetConfig: ApiAnalyticsDashboardWidgetConfig,
    urlParamsData: ApiAnalyticsWidgetUrlParamsData,
  ): Observable<ApiAnalyticsWidgetConfig> {
```
If you use the object type instead of separate parameters, it will allow evolutions in the future.

This can be done for all the methods that you have adapted to the new query params, including the `toQuery` 👍